README.md

```diff
@@ -1,2 +1,46 @@
 # Women-summit
 women's summit to highlight some of the problems facing women
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+- BOOTSTRAP
+
+## Live Demo
+
+[Live Demo Link](https://hanningtonem.github.io/Women-summit/)
+
+## Getting Started
+
+To get a local copy up and running:
+
+1. Clone this repository or download the Zip folder:
+
+**``git clone https://github.com/HanningtoneM/Women-summit.git``**
+
+2. Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd <folder>``**
+
+## Author
+
+👤 **hanningtone Monda**
+
+- Github: [@HanningtoneM](https://github.com/HanningtoneM)
+
+
+## Credits
+
+Template from this [Figma](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by [Microverse](https://bit.ly/MicroverseTN)
+
```

Comment:
- [ ] Kindly you can mention the name of the author [Cindy Shin](https://www.behance.net/adagio07) instead of Microverse.